src/Build/BuildCheck/Infrastructure/BuildAnalyzerConfigurationInternal.cs

```diff
@@ -19,8 +19,11 @@ public BuildAnalyzerConfigurationInternal(string ruleId, EvaluationAnalysisScope
     }
 
     public string RuleId { get; }
+
     public EvaluationAnalysisScope EvaluationAnalysisScope { get; }
+
     public BuildAnalyzerResultSeverity Severity { get; }
+
     public bool IsEnabled { get; }
```

Comment:
```suggestion
    public bool IsEnabled => Severity >= BuildAnalyzerResultSeverity.Suggestion;
```

If check doesn't have default in code nor explicit setting in editorconfig - it's considered disabled. None is as well disabled. Everything else is enabled.

Reply:
This is something that is being set during the creating of the instance. Based on the setting of editor config and the default values. 
Hence there is no need to set it here.

Reply:
Please let me know if it is not clear I'll adjust the code, thank you

Reply:
This is updated, thanks